### PR TITLE
Feature/faas 1509 category page styling

### PR DESCRIPTION
**Pull request by abogdanov-work**
## PR Description
This PR fixes the overall SCSS structure of the project eliminating a lot of internal theme issues

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: pages/Category.vue [hunk 1]
```diff
@@ -23,14 +23,14 @@
     />
     <LayoutSlot :slotName="slotName" />
     <div class="navbar section">
-      <div class="navbar__aside desktop-only">
+      <!--      <div class="navbar__aside desktop-only">
         <SfHeading
```
Comment (FlmobSpryker) [inline pages/Category.vue:27]: What is that part commented for?

File: assets/root.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  @include assign-color-variants(--c-secondary-variant, --_c-orange-secondary);
+
+  --badge-background: var(--_c-orange-primary);
+  --overlay-background: rgba(0, 0, 0, 0.3);
+  --price-old-color: #b2b2b2;
+  --price-special-color: var(--_c-orange-primary);
+  --price-regular-color: #333333;
+  --header-background: #e7eaee;
+  --top-bar-background: #d0d5dc;
+  //--overlay-z-index: 4;
```
Comment (tobi-or-not-tobi) [inline assets/root.scss:24]: can we drop this?

**Review by tobi-or-not-tobi — APPROVED** [review]
All good from my side, but lets answer the q from @ykravchuk-spryker

File: assets/root.scss (hunk 2 above)
Comment (abogdanov-work) [reply assets/root.scss:24]: this one will be needed anyway a bit later, I marked the place where z-index related fixes should be placed

**Pull request merged**